readme.md

```diff
@@ -575,6 +575,7 @@
 - [Crypto Currency Tools & Algorithms](https://github.com/kennethreitz/awesome-coins) — Digital currency where encryption is used to regulate the generation of units and verify transfers.
 - [Open Source Supporters](https://github.com/zachflower/awesome-open-source-supporters) - Companies that offer their tools and services for free to open source projects.
 - [Design Principles](https://github.com/robinstickel/awesome-design-principles) - Create better and more consistent designs and experiences.
+- [Tech for Good](https://github.com/TechforgoodCAST/awesome-techforgood) - A selection of projects, organisations and useful tools for social-impact tech.
```

Comment:
> Include a short description about the project/theme of the list. **It should not describe the list itself.**

Reply:
The repo name is also not very readable. Should be `awesome-techforgood` => `awesome-tech-for-good`.